server/views/pages/overview/partials/educationAndTrainingTab/educationAndTrainingTabContents.njk

```diff
@@ -40,9 +40,17 @@ The 'Training and eduction in prison' summary card is only shown if the short qu
           </div>
 
           <div class="govuk-summary-card__content">
+            {%- set createInductionUrl -%}
```

Comment:
The following changes all set the URL of the Create Induction journey to either the PLP or CIAG UI first page, depending on whether the feature toggle is set